buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/DistributedPerformanceTest.groovy

```diff
@@ -132,10 +131,24 @@ class DistributedPerformanceTest extends PerformanceTest {
             doExecuteTests()
         } finally {
             testEventsGenerator.release()
+            generatePerformanceReport()
             cleanTempFiles()
         }
     }
 
+    private void generatePerformanceReport() {
+        project.delete(reportDir)
```

Comment:
Do we need a `reportDir.mkdirs()` here?

Reply:
Seems not because `parentMkdirs` are used before rendering: https://github.com/gradle/gradle/blob/1a77a941b1d98e055cd803a776ae7f9deaf68c34/subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/results/FileRenderer.java#L31